schedule/virtualization/mau-virt-increments-installation.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: mau-virt-increments-installation
+description: >
+  Maintainer: qe-virt@suse.de
+  SLES 16.0 virtualization packages increment testing
+  All 4 platforms: x86_64, aarch64, s390x, ppc64le
+conditional_schedule:
+  boot_s390x:
+    ARCH:
+      's390x':
+        - installation/bootloader_zkvm
+  registration:
+    SCC_REGISTER:
+      'installation':
+        - console/suseconnect_scc
+  kvm:
+    ARCH:
+      # KVM tests are supported on x86_64, s390x
+      # aarch64/ppc64le: nested KVM not available in qemu backend, skip
```

Comment:
Hi Roy,
in qemu backend, actually nested is widely used for virtualization test, although even for x86, it is not officially supported. And this task, we actually mainly rely on nested to test our VT.
During sle16 test, we opened a ppc bug but did not have kvm enabled, dev asks us to add, and share that it will be more valuable to have kvm acceleration tested in our code. 
For aarch64, the test is qemu backend, so shall be in a even better situation than ppc to run kvm test.
Would you have a try to enable the kvm test for aarch64 and ppc?

Reply:
I will submit another PR to fix it.
firstly enable kvm module for aarch64 and ppc platform